client/signup/steps/dss/index.jsx

```diff
@@ -120,6 +123,22 @@ export default React.createClass( {
 		);
 	},
 
+	renderTheme( theme ) {
+		return (
+			<DssThemeThumbnail
+				key={ theme.name }
+				themeName={ theme.name }
+				themeSlug={ theme.slug }
+				themeRepoSlug={ 'pub/' + theme.slug }
+				isLoading={ this.state.isLoading }
+				dssImage={ this.state.dssImage }
+				lastSearchTerm={ this.state.lastSearchTerm }
+				markupAndStyles={ this.state.markupAndStyles[ 'pub/' + theme.slug ] }
+				renderComplete={ this.state.renderComplete }
+				{ ...this.props }/>
```

Comment:
I'm in favor of putting the `/>` (and even the `>` for tags with children) on the following line:

- Like `);` and `};` below, it denotes the end of a block
- Easier to rearrange props, like trailing `,` in arrays

That's just my opinion and attempt to change your personal preference :) If you go with the style here, this needs a space in between `}` and `/>`.